src/herder/Upgrades.cpp

```diff
@@ -55,19 +73,15 @@ load(Archive& ar, stellar::Upgrades::UpgradeParameters& o)
     o.mUpgradeTime = stellar::VirtualClock::from_time_t(t);
     ar(make_nvp("version", o.mProtocolVersion));
     ar(make_nvp("fee", o.mBaseFee));
-    ar(make_nvp("maxtxsize", o.mMaxTxSize));
+    // maxtxsize has been renamed to maxtxsetsize. We need to try safely loading
+    // both in order to provide backwards compatibility.
+    safeLoadValue(ar, "maxtxsize", o.mMaxTxSetSize);
```

Comment:
I suspect it shouldn't matter since if something has been broken before the change, it would be unlikely for it to pickup the new change. But I guess there isn't much value in having backwards compatibility logic here, given that IIUC this is just an internal serialization format. Rolled back all the changes and left the field named `maxtxsize`.